Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindow.cs

```diff
@@ -30,7 +30,8 @@ static InputActionsEditorWindow()
         }
 
         static readonly Vector2 k_MinWindowSize = new Vector2(650, 450);
-
+        // For UI testing purpose
+        internal InputActionAsset currentAssetInEdition => m_AssetObjectForEditing;
```

Comment:
Is this name a typo?
currentAssetInEdition 

Should it be ...
currentAssetInEditior